tests/Provider/Doctrine/Issues/Issue115Test.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DH\Auditor\Tests\Provider\Doctrine\Issues;
+
+use DH\Auditor\Provider\Doctrine\Auditing\Transaction\TransactionProcessor;
+use DH\Auditor\Provider\Doctrine\DoctrineProvider;
+use DH\Auditor\Provider\Doctrine\Model\Transaction;
+use DH\Auditor\Provider\Doctrine\Persistence\Reader\Reader;
+use DH\Auditor\Provider\Doctrine\Service\AuditingService;
+use DH\Auditor\Provider\Doctrine\Service\StorageService;
+use DH\Auditor\Tests\Provider\Doctrine\Fixtures\Issue115\DummyEntity;
+use DH\Auditor\Tests\Provider\Doctrine\Fixtures\Issue115\DummyEnum;
+use DH\Auditor\Tests\Provider\Doctrine\Traits\Schema\DefaultSchemaSetupTrait;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @internal
+ *
+ * @small
+ */
+final class Issue115Test extends TestCase
+{
+    use DefaultSchemaSetupTrait;
+
+    public function testIssue115(): void
+    {
+        if (\PHP_VERSION_ID < 80000) {
```

Comment:
`BackedEnum` are only available in PHP >= 8.1.0 (see https://php.watch/versions/8.1)

Reply:
Done